examples/next-prisma-todomvc/src/pages/[filter].tsx

```diff
@@ -70,13 +70,12 @@ function ListItem({ task }: { task: Task }) {
 
   const editTask = trpc.useMutation('todo.edit', {
     async onMutate({ id, data }) {
-      await utils.cancelQuery(['todo.all']);
-      const allTasks = utils.getQueryData(['todo.all']);
+      await utils['todo.all'].cancel();
```

Comment:
I really think this ought to be

```suggestion
      await utils.todo.all.cancel();
```

Any reason why you opted for not doing it that way?